tests/components/ozw/test_websocket_api.py

```diff
@@ -56,3 +58,75 @@ async def test_websocket_api(hass, generic_data, hass_ws_client):
     assert result["received_packets"] == 3594
     assert result["received_dup_packets"] == 12
     assert result["received_unsolicited"] == 3546
+
+    # Test node metadata
+    await client.send_json({ID: 8, TYPE: "ozw/node_metadata", NODE_ID: 39})
+    msg = await client.receive_json()
+    result = msg["result"]
+    assert result["metadata"]["ProductPic"] == "images/aeotec/zwa002.png"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)